plugins/modules/lxd_container.py

```diff
@@ -239,7 +239,9 @@
 
     - name: Install python in container
       delegate_to: mycontainer
-      ansible.builtin.raw: apt-get install -y python
+      ansible.builtin.apt:
+        name: python3
+        state: present
```

Comment:
This change is not compatible with what the example is trying to achieve. The example installs Python if it is not present (checked in the previous task). Using the `ansible.builtin.apt` module only works if Python is already installed.

Reply:
hmm ok, then this should maybe be documented as well. 
Ubuntu comes with python now preinstalled, afaik. (it also was fine during my test)

Reply:
Standard Ubuntu installations probably do, but Ubuntu based container images might not.

Reply:
AFAIK Python3 is now standard for all Ubuntu images, even minimal. https://packages.ubuntu.com/focal/ubuntu-minimal. It seems some system packages need it.

If this should be an example of installing a package, I propose to install something more "exotic" like `ruby` or something similar.

Reply:
I'm not sure that just because the OS distribution ubuntu-minimal contains a package it means that *every* container image with that OS in the name also includes that package. For example, the official Ubuntu Docker container (https://hub.docker.com/_/ubuntu) does *not* have Python installed.

In any case, this example is *not* about installing a package, but *explicitly* about installing Python, which is needed to run Ansible modules. So either remove the whole example playbook, or simply keep it that way.